src/BaseTokenWrapper.sol

```diff
@@ -103,6 +103,14 @@ abstract contract BaseTokenWrapper is Ownable, IBaseTokenWrapper {
     return _withdrawToken(amount, to, aTokenOut);
   }
 
+  /// @inheritdoc IBaseTokenWrapper
+  function borrowToken(uint256 amount, address to) external virtual {
```

Comment:
add support to borrow by using sig-based delegated credit

Reply:
added in 6915b969897ebe65b6ee957bfd19abffa0beb009